test/unit/http_test.lua

```diff
@@ -242,6 +244,35 @@ local test_mixed_cases = {
             "http://127.0.0.1:8081/metrics4",
         },
     },
+    ["listen_tls"] = {
+        cfg = {
+            http = {
+                {
+                    listen = 8081,
+                    ssl_key_file = fio.pathjoin("test", "ssl_data", "server.key"),
+                    ssl_cert_file = fio.pathjoin("test", "ssl_data", "server.crt"),
+                    endpoints = {
+                        {
+                            path = "/metrics1",
+                            format = "json",
+                        },
+                        {
+                            path = "/metrics2",
+                            format = "prometheus",
+                        },
+                    },
+                },
+            },
+        },
+        expected_json = {
+            "https://localhost:8081/metrics1",
+        },
+        expected_prometheus = {
+            "https://localhost:8081/metrics2",
+        },
```

Comment:
Since we test a TLS here, it could be simplified:
```suggestion
                    endpoints = {
                        {
                            path = "/metrics1",
                            format = "json",
                        },
                    },
                },
            },
        },
        expected_json = {
            "https://localhost:8081/metrics1",
        },
```

Reply:
Simplified.